src/components/HomepageBanner.js

```diff
@@ -0,0 +1,301 @@
+          padding-top: 0;
+          max-height: none;
+          height: 500px;
+        }
+        @media screen and (max-width: 320px) {
+          height: 460px;
+        }
+      `}
+    >
+      <div
+        css={css`
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+            display: flex;
+            flex-direction: column;
+          }
+        `}
+      >
+        <picture>
+          <source
+            srcSet={bannerForegroundMobile}
+            media={`(max-width:${BREAKPOINTS.SINGLE_COLUMN})`}
+          />
+          <img
+            src={bannerForeground}
+            alt="banner foreground"
+            css={css`
+              position: relative;
+              height: 100%;
+              max-height: 423px;
+              z-index: 1;
+              @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+                max-height: 100%;
+                width: 100%;
+                top: 150px;
+              }
+              @media screen and (max-width: ${BREAKPOINTS.MOBILE}) {
+                top: 140px;
+              }
+            `}
+          />
+        </picture>
+        <picture>
+          <source
+            srcSet={relicsAtWorkMobile}
+            media={`(max-width:${BREAKPOINTS.SINGLE_COLUMN})`}
+          />
+          <img
+            src={relicsAtWork}
+            alt="relics at work"
+            css={css`
+              position: absolute;
+              top: 0;
+              right: 0;
+              width: 50%;
+              @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+                width: 100%;
+                top: -10%;
+              }
+              @media screen and (max-width: ${BREAKPOINTS.MOBILE}) {
+                top: 0;
+              }
+            `}
+          />
+        </picture>
+        <div
+          css={css`
+            position: absolute;
+            top: 10%;
+            left: 5%;
+            z-index: 2;
+            width: 48%;
+
+            @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+              width: 85%;
+              top: 180px;
+              left: 0;
+              padding: 0 1rem;
+              height: 40%;
+              display: flex;
+              flex-direction: column;
+              justify-content: space-between;
+              p {
+                display: block;
+                font-size: max(3.5vw, 11px);
+                padding-right: 1rem;
+              }
+            }
+            @media screen and (max-width: ${BREAKPOINTS.MOBILE}) {
+              top: 155px;
+              width: 100%;
+              height: 30%;
+              p {
+                padding-right: 1.5rem;
+                margin-bottom: 0;
+              }
+            }
+          `}
+        >
+          <h1
+            css={css`
+              color: white;
```

Comment:
The theme has a color that's pretty close to the design for this header. We can probably remove this.

```suggestion
```